package.json

```diff
@@ -26,7 +26,7 @@
     "theme": "dark"
   },
   "engines": {
-    "vscode": "^0.10.1"
+    "vscode": "^1.6.0"
```

Comment:
`onWillSaveTextDocument` was introduced in `1.6.0` so that's why we need engine bump.